en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -0,0 +1,102 @@
+
+#### WSO2 Integration Studio
+
+The following features and improvements are available for <b>WSO2 Integration Studio</b> EI 7.1.0:
+
+Improvements to [<b>Connector</b>](../../references/connectors/connectors-overview):
+
+-   Revamped connector story with improved user experience
+-   Redesigned connector configuration view
+-   New externalizable connection configuration view for connectors
+<!--   Improved CI/CD story with WSO2 Micro Integrator-->
+
+Improvements to [<b>integration project</b>]((../../develop/create-integration-project):
+
+-   Maven multi module support for the integration project
+<!-- -   Embedded maven support -->
+-   New maven profiles for project build  
+-   Improved project import/export functionality
+<!-- -   Capability to import existing projects into maven multi module project -->
+
+Improvements to [<b>Docker project</b>](../../develop/create-docker-project):
+
+-   Improved Docker project
+-   Capability to push images to a remote repository via the Docker exporter
+
+Improvements to the [<b>embedded Micro Integrator</b>](../../develop/using-embedded-micro-integrator):
+
+-	WSO2 Micro Integrator 1.2.0 as the embedded server runtime
+-	Hot deployment capability for the embedded Micro Integrator
+-   Capability to configure the embedded Micro Integrator via the tool
+-   Embedded WSO2 EI monitoring dashboard to list details of the deployed services
+
+Improvements to <b>debugging and testing</b>:
+
+-   Capability to [run and debug synapse configurations](../../develop/debugging-mediation)
+<!-- -   Listing deployed services via the embedded CLI tool -->
+-   Improved [Synapse Unit testing framework](../../develop/creating-unit-test-suite)
+-   Test failure observability improvements
+<!-- -   Synapse template testing support -->
+<!-- -   Synapse sequence testing improvements -->
+
+Improvements to <b>artifact development</b>:
+
+-   Improved [data mapper mediator](../../references/mediators/data-Mapper-Mediator)
+<!--  -   AI-based data mapper improvements -->
+<!--  -   Data mapper functionality and UI improvements -->
+-   Improved [data services editor](../../develop/creating-artifacts/data-services/creating-data-services)
+<!--  -   Improved editor theme -->
+<!-- -   RCP improvements -->
+
+Other improvements:
+
+-   New WSO2 Integration Studio theme
+-   New 'Getting Started' experience
+<!-- -   New update notification functionality -->
+
+#### Streaming Integrator
+
+The following features and improvements are available for the <b>Streaming Integrator</b> of EI 7.1.0:
+
+-   Extension Installer CLI tool to install Siddhi extensions via the command line.
+-   Error Store in which the events with errors can be browsed. This also allows events with certain types of errors to be replayed.
+-   ETL wizard that allows you to design ETL applications without writing code.
+-   New monitoring dashboards to analyze ETL flows.
```

Comment:
For more information, see [Monitoring Statistics via Grafana Dashboards](https://ei.docs.wso2.com/en/7.1.0/streaming-integrator/admin/setting-up-grafana-dashboards/).